packages/eslint-config-typescript/index.js

```diff
@@ -113,4 +113,14 @@ module.exports = {
     'space-infix-ops': 'off',
     '@typescript-eslint/space-infix-ops': ['error'],
   },
+  overrides: [
+    {
+      files: ['test/**'],
```

Comment:
```suggestion
      files: ['*.test.*'],
```
Acho que é melhor assim, pq nem todos os projetos vão ter todos seus arquivos de teste dentro da pasta `test/`, mas acho que todos os testes tem sufixo de `.test.(js|ts)`

Reply:
Tem alguns projetos que usam spec `['*.test.*', '*.spec.*']` :sweat_smile: